packages/e2e-test-utils/src/wp-data-select.js

```diff
@@ -5,9 +5,9 @@
  * @param {string}    selector   Selector to exectute e.g: getBlocks.
  * @param {...Object} parameters Parameters to pass to the selector.
  *
- * @return {?Object} Result of querying.
+ * @return {Promise<?Object>} Result of querying.
  */
-export function wpDataSelect( store, selector, ...parameters ) {
+export async function wpDataSelect( store, selector, ...parameters ) {
```

Comment:
Great catch, `page.evaluate` that is used is async:

https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#pageevaluatepagefunction-args

It might make some tests more stable. /cc @youknowriad

Reply:
I don't  think it makes any difference to add "async" here since page.evalute returns a promise already

Reply:
It doesn't make any difference but it's explicit that returns a `Promise`. So someone would know they have to `await`.

Reply:
In any case, the function signature was wrong, so this is a good catch.